kibana/Dockerfile

```diff
@@ -1,55 +1,50 @@
-FROM rhscl/nodejs-6-rhel7
+FROM rhscl/nodejs-10-rhel7 AS builder
```

Comment:
Use the ubi base image since you aren't using nodejs in the builder phase - I guess there isn't anything to build?